config/riscv-cva6.config

```diff
@@ -66,26 +72,27 @@
     <default_class category="bit_field" class="BitField"/>
   </default_classes>
   <register_files>
-    <register_file file="riscv/arch_data/app_registers.xml"/>
-    <register_file file="riscv/arch_data/system_registers.xml"/>
+    <register_file file="riscv/arch_data/reg/app_registers_rv64.xml"/>
+    <register_file file="riscv/arch_data/reg/system_registers_rv64.xml"/>
   </register_files>
   <choices_files>
-    <choices_file file="riscv/arch_data/operand_choices.xml"/>
-    <choices_file file="riscv/arch_data/general_choices.xml"/>
-    <choices_file file="riscv/arch_data/register_field_choices.xml"/>
-    <choices_file file="riscv/arch_data/paging_choices.xml"/>
-    <choices_file file="riscv/arch_data/dependence_choices.xml"/>
-    <choices_file file="riscv/arch_data/system_register_choices.xml"/>
+    <choices_file file="riscv/arch_data/general/operand_choices.xml"/>
+    <choices_file file="riscv/arch_data/general/general_choices.xml"/>
+    <choices_file file="riscv/arch_data/reg/register_field_choices_rv64.xml"/>
+    <choices_file file="riscv/arch_data/paging/paging_choices_sv48.xml"/>
+    <choices_file file="riscv/arch_data/general/dependence_choices.xml"/>
+    <choices_file file="riscv/arch_data/reg/system_register_choices_rv64.xml"/>
   </choices_files>
   <paging_files>
-    <paging_file file="riscv/arch_data/page_tables.xml"/>
+    <paging_file file="riscv/arch_data/paging/page_tables_sv48.xml"/>
```

Comment:
This configuration has always been set to use SV48. However, I very briefly scanned through the CVA6 source code. Does CVA6 actually support SV48? I only found evidence of implementations for SV32 and SV39.

Reply:
AFAIK, the CVA6 does not support SV48.  @JeanRochCoulon, can you confirm this?

Reply:
Indeed, CVA6 does not support SV48 (till now!). CVA6 supports SV0 (no MMU), SV32 and SV39.